iotilecore/iotile/core/scripts/virtualdev_script.py

```diff
@@ -162,6 +170,21 @@ def main(argv=None, loop=SharedLoop):
     return 0
 
 
+async def supervise_server(server, timeout=0.5):
```

Comment:
Maybe not necessary, but what type of object does `server` expect? From contest I gather it is supposed to be `AbstractDeviceServer` and derived classes.

Reply:
annotation added